README.md

```diff
@@ -1,2 +1,24 @@
-# ms-office-macro-nuke
-Tool to Purge and Remove all Macro And Dynamic Content from MS Office File
+![MS Office Macro Bleach](docs/images/bleach.jpg)
+
+
+# MS Office Macro Bleach
+
+A command-line tool designed to detect and purge any and all macros and dynamic content from MS Office files.
+
+It currently supports '.docm' and'.docx' files and will continue to expand its file support with every released version. The next file types to be supported will be '.pdf'' and '.doc'.
+
+
+## Problem
+
+VBA and OLE content in MS Office files can, and have sometimes been made to, act as vehicles for malware delivery.
+
+Microsoft has previously attempted to protect users from macros by disabling them by default.  However, anybody is able to enable macros in an MS Office file before sending them on to a potential victim.
+
+This Python tool aims to detect and remove any of this potentially malicious content from given files.
+
+
+## Solution
+
+A command-line program written in modern Python (3.10+) that is capable of locating and removing macros and dynamic content from a variety of files.
+
+It should support all the common Office Open XML formats (e.g. '.pptx', .docx', '.xlsx', etc.) as well as the 'legacy' MS binary file formats, like '.doc' and '.ppt'.
```

Comment:
You also need to add instructions here on the usage, the command lines to trigger this. 
Look into using the scripts section of poetry to create a linux command like
docubleach file.doc

This would, when installed the package create a command called docubleach that on the backend, will call your python file and some function within. 

We will then add integration to build and publish this package via pipy so that people can do something like

pip install docubleach
docubleach file.doc

To execute the tool the clean the files.

Reply:
You can also add a section of how this script could be integrated in other solutions (although we might decide to do this later) Some ideas are:

1) Integrate it with Postfix (Email Pipeline), so that all attachment emails that match a list of extensions are run through docubleach.
1) Integrate it with Squid (Web Gateway Pipeline), so that all attachment emails that match a list of extensions are run through docubleach.

Reply:
Thanks for the feedback. I'm going to start with fixing the textual changes, then create a new testing branch to test all OOXML so I can write accurately if it works on those formats too.

The way its set up to run now is 'poetry run bleach FILE' - I will investigate how to make this just a command line command called 'docubleach' in accordance with poetry.

Reply:
The nice thing to do it like this, is you dont require your users to know about poetry, and if you do poetry build and publish the wheel package on pipy they dont even need poetry.
For the future I foresee we build this package in a CI and publish it in pipy and we also publish a deb and rpm package and publish those, so that people can do something simple like
apt-get install docubleach and make use of the tool.

Reply:
Good points! It needs to be easy to use and accessible for anybody.